static/styles/zulip.scss

```diff
@@ -2243,6 +2247,16 @@ li .message_inline_image img {
     float: none;
 }
 
+.youtube-video .fa-play::before {
+    position: absolute;
+    margin: 32px 0 0 45px;
+    padding: 5px 8px 5px 10px;
+    font-size: 12px;
+    border-radius: 4px;
+    background: #212121;
+    color: #fff;
+}
```

Comment:
Can we use HSLA for all the colors?  That's our standard across the app at this point (there are plans to introduce a lint rule for it).  Also, should we perhaps do the centering a cleaner way?  

@synicalsyntax FYI.

Reply:
Yeah, I forgot, will update to HSLA.